tensorflow/compiler/xla/service/gpu/cudnn_batchnorm_rewriter.cc

```diff
@@ -195,15 +265,57 @@ Status Visitor::HandleBatchNormGrad(HloInstruction* batch_norm) {
 
   std::vector<HloInstruction*> operands(batch_norm->operands().begin(),
                                         batch_norm->operands().end());
+  bool is_convert_inserted = false;
```

Comment:
Would it be too much work to split out `HandleBatchNormGrad` into a separate PR?  That way we'll be able to isolate performance issues more easily.

Reply:
Will submit separate PR once https://github.com/tensorflow/tensorflow/pull/32887 gets merged.